core/main.go

```diff
@@ -76,15 +76,31 @@ func stopRunningServices(sdk *enginesdk.SDK) error {
 	return errs.ErrorOrNil()
 }
 
+func loadOrGenConfigAccount(kb *cosmos.Keybase, cfg *config.Config) (keys.Info, error) {
+	exist, err := kb.Exist(cfg.Account.Name)
+	if err != nil {
+		return nil, err
+	}
+	if exist {
+		return nil, nil
+	}
+	logrus.WithField("module", "main").Warn("Config account not found. Generating one for development...")
+	mnemonic, err := kb.NewMnemonic()
```

Comment:
We should have the mnemonic in the config (empty by default) and if empty generate a new one, otherwise use the one in the config.

This will let us start the engine with a pre-determined address and we don't need to rely on a generated address.

Reply:
issue created https://github.com/mesg-foundation/engine/issues/1470